backend/manager/modules/utils/src/main/java/org/ovirt/engine/core/utils/ovf/OvfReader.java

```diff
@@ -975,6 +978,44 @@ private void fixDiskVmElements() {
         }
     }
 
+    protected void readNumaNodeListSection(XmlNode section) {
+        XmlNodeList list = selectNodes(section, NUMA_NODE);
+        List<VmNumaNode> vmNumaNodes = new ArrayList<>();
+        vmBase.setvNumaNodeList(vmNumaNodes);
```

Comment:
I think it would be a bit easier to read it you do that at the end, after adding the elements to the list (sure, it works this way as well because we pass it by reference so only because of readability)